src/v2/examples/themes.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Themes
+type: examples
+is_new: true
+order: 13
+---
+> With the examples below built by our partners from [Creative Tim](https://creative-tim.com?affiliate_id=116187) you can see how a real world application is built, the technology stack behind it and how most of the concepts you've learned so far apply in a real world application. 
+
+{% raw %}
+<div id="themes-example" class="themes-grid">
+   <div v-for="product in products" :key="product.name" class="item-preview">
+     <a class="item-preview-img" :href="`https://www.creative-tim.com/product/${product.name}?affiliate_id=${affiliateId}`" rel="nofollow">
+     <img :src="`https://raw.githubusercontent.com/creativetimofficial/public-assets/master/${product.name}/${product.name}.jpg`" :alt="`${product.title} - ${product.description}`"></a>
+     <div class="item-preview-title-container">
+      <h3 class="item-preview-title" :class="{'free': product.free}">{{product.title}}</h3>
```

Comment:
Somehow the scrolling-to-header in the sidebar links isn't working for these `h3` titles